app/viewmodels/models/TextAreaViewModel.scala

```diff
@@ -16,6 +16,12 @@
 
 package viewmodels.models
 
+import viewmodels.DisplayMessage.Message
+
 case class TextAreaViewModel(
-  rows: Int = 5
-)
+  rows: Int = 5,
+  hint: Option[Message] = None
```

Comment:
It shouldn't break anything, but double check please, as it is used in multiple places. Check a11y too.